There are no Android realeases in few last versions

### Describe the bug


https://github.com/CleverRaven/Cataclysm-DDA/releases

### Attach save file

-

### Steps to reproduce

-

### Expected behavior

Android releases need to be.

### Screenshots

_No response_

### Versions and configuration

-

### Additional context

_No response_

Seems to be fallout from #74796, since we're still lacking android CI. Should be an easy fix, though.

```
  /home/runner/work/Cataclysm-DDA/Cataclysm-DDA/android/app/jni/src/../../../../src/sdltiles.cpp:2756:29: error: no matching function for call to 'can_interact_at'
                          if( can_interact_at( ACTION_PICKUP, pos ) ) {
                              ^~~~~~~~~~~~~~~
  /home/runner/work/Cataclysm-DDA/Cataclysm-DDA/android/app/jni/src/../../../../src/action.h:645:6: note: candidate function not viable: no known conversion from 'const tripoint' to 'const tripoint_bub_ms' (aka 'const coord_point<tripoint, coords::origin::reality_bubble, coords::ms>') for 2nd argument
  bool can_interact_at( action_id action, const tripoint_bub_ms &p );
       ^
```

--- a/src/sdltiles.cpp
+++ b/src/sdltiles.cpp
@@ -2753,7 +2753,7 @@ static void CheckMessages()
                         }
 
                         // Check for actions that work on nearby tiles and own tile
-                        if( can_interact_at( ACTION_PICKUP, pos ) ) {
+                        if( can_interact_at( ACTION_PICKUP, tripoint_bub_ms( pos ) ) ) {
                             actions.insert( ACTION_PICKUP );
                         }
                     }

